workloads/templates/workload-mastervertical-script-cm.yml.j2

```diff
@@ -83,10 +83,7 @@ data:
       export AZURE_AUTH_LOCATION=/tmp/azure_auth
     fi
     start_time=$(date +%s)
-    export es={{ snafu_es_host }}
-    export es_port={{ snafu_es_port }}
-    export es_index={{ snafu_es_index_prefix }}
-    VIPERCONFIG=/tmp/mastervertical.yaml python /tmp/snafu/run_snafu.py -t cl scale-ci --cl-output True --dir "${result_dir}" -p openshift-tests | tee "${result_dir}/clusterloader.txt"
+    VIPERCONFIG=/tmp/mastervertical.yaml openshift-tests run-test "[Feature:Performance][Serial][Slow] Load cluster concurrently with templates [Suite:openshift]" | tee "${result_dir}/clusterloader.txt"
```

Comment:
Do you think [Feature:Performance][Serial][Slow] has something to do with cluster loader taking a bit longer to create objects?